src/eip4824.ts

```diff
@@ -0,0 +1,157 @@
+import express from 'express';
+import { getSpace } from './helpers/spaces';
+import db, { sequencerDB } from './helpers/mysql';
+
+const router = express.Router();
+
+router.get('/:space', async (req, res) => {
+  let space: any = {};
+  const protocol = req.get('X-Forwarded-Proto') || req.protocol;
+  const host = req.get('X-Forwarded-Host') || req.get('host');
+  const baseUrl = `${protocol}://${host}${req.originalUrl}`;
+
+  try {
+    space = await getSpace(req.params.space);
+
+    if (!space.verified) return res.status(404).json({ error: 'INVALID' });
+  } catch (e) {
+    console.log(e);
+    return res.status(500).json({ error: 'NOT_FOUND' });
```

Comment:
Even if there is some error with `getSpace` or DB, we return `NOT_FOUND`

Reply:
Yea this is annoying, when there is an error we don't know if it's because a space doesnt exist or if the MySQL request failed.